Sources/MapKitSearchViewController.swift

```diff
@@ -759,3 +824,12 @@ class PlaceAnnotation: NSObject, MKAnnotation {
         subtitle = nil
     }
 }
+
+
+extension CLLocationCoordinate2D: Equatable {
```

Comment:
Would it make sense to put this into the `Extensions` folder. Just for structure and moving the responsibility to that folder.